compiler/rustc_session/src/config.rs

```diff
@@ -2318,7 +2346,11 @@ pub fn build_session_options(matches: &getopts::Matches) -> Options {
         }
 
         // Only use this directory if it has a file we can expect to always find.
-        if candidate.join("library/std/src/lib.rs").is_file() { Some(candidate) } else { None }
+        if candidate.join("library/std/src/lib.rs").is_file() {
+            Some(candidate)
+        } else {
+            None
+        }
```

Comment:
Looks like this formatting was done by an incorrect version of rustfmt.
The correct version is taken from `./build/cache` and is used if you are formatting with `x.py fmt`, for example.